erc4626/MorphoVaults/V1.1 Gauntlet.md

```diff
@@ -0,0 +1,42 @@
+# ERC4626 Vault: `MetaMorphoV1_1`
+
+## Details
+- Reviewed by: @mkflow27
+- Checked by: 
+- Deployed at:
+    - [ethereum:https://etherscan.io/address/0x1e6ffa4e9F63d10B8820A3ab52566Af881Dab53c](https://etherscan.io/address/0x1e6ffa4e9F63d10B8820A3ab52566Af881Dab53c)
+    - [ethereum:https://etherscan.io/address/0x701907283a57FF77E255C3f1aAD790466B8CE4ef](https://etherscan.io/address/0x701907283a57FF77E255C3f1aAD790466B8CE4ef)
+
+- Audit report(s):
+    - [Security Reviews & Formal Verifications](https://github.com/morpho-org/metamorpho-v1.1/tree/main/audits)
+    - [MetaMorpho Spearbit Audit](https://github.com/morpho-org/metamorpho/blob/main/audits/2023-11-14-metamorpho-cantina-managed-review.pdf)
+
+## Context
+A 4626 Vault which wraps underlying tokens in MetaMorpho vaults in order for vault curators to earn liquidity providers additional yield on their assets.
+
+## Review Checklist: Bare Minimum Compatibility
+Each of the items below represents an absolute requirement for the Rate Provider. If any of these is unchecked, the Rate Provider is unfit to use.
+
+- [x] Tests based on the [balancer-v3-monorepo](https://github.com/balancer/balancer-v3-monorepo/tree/main/pkg/vault/test/foundry/fork) pass for the given ERC4626 vaults.
+- [x] The required Vault implements the required operational ERC4626 Interface
+
+## Review Checklist: Common Findings
+Each of the items below represents a common red flag found in Rate Provider contracts.
+
+If none of these is checked, then this might be a pretty great Rate Provider! If any of these is checked, we must thoroughly elaborate on the conditions that lead to the potential issue. Decision points are not binary; a Rate Provider can be safe despite these boxes being checked. A check simply indicates that thorough vetting is required in a specific area, and this vetting should be used to inform a holistic analysis of the Rate Provider.
+
+### Administrative Privileges
+- [] The ERC4626 Vault is upgradeable.
```

Comment:
The [MetaMorphoV1_1](https://vscode.blockscan.com/ethereum/0xbeef7959ae71d4e45e1863dae0b94c35244af816) contract doesnt seem to inherit from any contracts that would allow for upgrade-ability